spec/plans/prepare.fmf

```diff
@@ -19,6 +19,16 @@ description: |
     individual tests order ``70`` is used, for recommended
     packages it is ``75``.
 
+    .. note::
+
+        If you want to use the ``prepare`` step to generate data
+        files for the testing in the ``exeute`` step, you need to
+        move or copy them into ``${TMT_TEST_DATA}`` directory. Only
+        files in this directory are pulled back to the local machine
```

Comment:
```suggestion
        files in this directory are guaranteed to be preserved.
```

I would probably skip the rest of the paragraph as those are rather implementation details, not interesting for the user and hard to explain precisely/correctly in a short sentence.